ouroboros-consensus-shelley-test/src/Test/ThreadNet/Infra/Shelley.hs

```diff
@@ -100,7 +100,7 @@ coreNodeKeys CoreNode{cnGenesisKey, cnDelegateKey, cnStakingKey, cnVRF, cnKES}
           , Gen.AllIssuerKeys
             { Gen.cold = mkDSIGNKeyPair cnDelegateKey
             , Gen.vrf  = mkVRFKeyPair cnVRF
-            , Gen.hot  = [(SL.KESPeriod 100, mkKESKeyPair cnKES)]
+            , Gen.hot  = [(SL.KESPeriod 0, mkKESKeyPair cnKES)]
             , Gen.hk   = SL.hashKey (SL.VKey $ deriveVerKeyDSIGN cnDelegateKey)
             }
```

Comment:
Are KES or VRF keys even needed to generate transactions? (That's the only thing `coreNodeKeys` is used for.) Maybe this type is used for generating more things than transactions. Try replacing things with `error` to see whether they are actually used.